docs/en/stack/ml/df-analytics/dfanalytics-limitations.asciidoc

```diff
@@ -40,6 +40,17 @@ You cannot update {dfanalytics-cap} configurations. Instead, delete the
 dedicated for {ml} processes. For general {ml} settings, see 
 {ref}/ml-settings.html[{ml-cap} settings in {es}].
 
+[float]
+[[dfa-time-limitations]]
+=== {dfanalytics-jobs-cap} runtime may vary
+
+The runtime of the {dfanalytics-jobs} depends on numerous factors, such as the 
+size of the dataset, the type of analytics, the number of fields that are included 
+in the analysis, the {ref}/ml-dfanalytics-resources.html#ml-hyperparameter-optimization[hyperparameters] used, and so on. For this reason, a 
```

Comment:
Here I think we should also talk about the type of the fields that are included in the analysis. For example, I think @tveasey has mentioned that running an analysis on a dataset with many numerical fields will take longer than running an analysis on a dataset that is purely composed of categorical fields. 
Also, we should mention that if any hyperparameters are supplied (meaning we do not have to run hyperparameter search), then this will lower the runtime (please confirm this @valeriy42 - thank you! :) )

Reply:
This is true :) The number of steps for  hyperparameter search is proportional to the number of undefined hyperparameters. Looking over some details, this means that going from from 4 hyperparameters to 3 would effectively reduce the time by 25%.